docs/static/playground/playground.ts

```diff
@@ -0,0 +1,311 @@
+// @ts-ignore
+const STORAGE_KEY = "SAVED_SAMPLE";
+const ENDPOINT_KEY = "SAVED_ENDPOINT";
+const existing = localStorage.getItem(STORAGE_KEY)
+
+// @ts-ignore
+const editor = monaco.editor.create(document.getElementById("container"), {
+    value: existing || "",
+    language: "typescript",
+    minimap: { enabled: false }
+});
+
+interface TargetInfo {
+    name: string;
+    id: string;
+    endpoints: TargetEndpoint[];
+}
+
+interface TargetEndpoint {
+    name: string;
+    url: string;
+}
+
+interface SampleInfo {
+    chapter: string;
+    name: string;
+    id: string;
+    path: string;
+    js?: string;
+}
+
+// interface defined in pxteditor\workspace.ts
+interface Project {
+    header?: any;
+    text?: { [ key: string ]: string };
+}
+
+const targets: TargetInfo[] = [
+    {
+        name: "Arcade",
+        id: "arcade",
+        endpoints: [
+            {
+                name: "beta",
+                url: "https://arcade.makecode.com/beta?controller=1"
+            },
+            {
+                name: "released",
+                url: "https://arcade.makecode.com?controller=1"
+            }
+        ],
+    }, {
+        name: "Minecraft",
+        id: "minecraft",
+        endpoints: [
+            {
+                name: "beta",
+                url: "https://minecraft.makecode.com/beta?ipc=1&inGame=1&controller=1"
+            },
+            {
+                name: "released",
+                url: "https://minecraft.makecode.com?ipc=1&inGame=1&controller=1"
+            }
+        ],
+    }, {
+        name: "Adafruit",
+        id: "adafruit",
+        endpoints: [
+            {
+                name: "beta",
+                url: "https://makecode.adafruit.com/beta?controller=1"
+            },
+            {
+                name: "released",
+                url: "https://makecode.adafruit.com?controller=1"
+            }
+        ],
+    }, {
+        name: "Micro:bit",
```

Comment:
```suggestion
        name: "micro:bit",
```

should probably follow standard formatting for their name?